src/main/kotlin/com/openlattice/mechanic/upgrades/Version.kt

```diff
@@ -39,5 +39,6 @@ enum class Version(val value: Long) {
     V2019_08_26(11L),
     V2019_09_15(12L),
     V2019_10_03(13L),
-    V2019_11_21(14L)
+    V2019_11_21(14L),
+    V2019_12_12(15L)
```

Comment:
the trailing comma is the oxford comma of programming.

Reply:
😢

Reply:
If we're going all in on the trailing commas, can we update our autoformatter to insert them?